lib/types/SSI.types.ts

```diff
@@ -251,42 +251,36 @@ export class InternalVerifiableCredentialJwt extends InternalCredentialJWT {
 
 export type InternalVerifiableCredential = InternalVerifiableCredentialJsonLD | InternalVerifiableCredentialJwt;
 
-export interface Credential {
```

Comment:
Why is the Credential interface still exposing both worlds? 

The VerifiableCredential approach you took with the seperation into Jwt and JsonLD is nice. 
I rather see the same thing for Credential.

So basically move almost everything over from JwtVerifiableCredential into JWTCredential with the exception of the proof. Similar for the JsonLdVerifiableCredential into a JsonLdCredential, with the exception of the proof. Then JWTVerifiableCredential extends JWTCredential and JsonLdVerifiableCredential extends JsonLdCredential.

Then have 2 types:
Credential = JsonLdCredential | JwtCredential
VerifiableCredential = JsonLdVerifiableCredential | JwtVerifiableCredential

Reply:
I removed `Credential` interface. we just have this part `VerifiableCredential = JsonLdVerifiableCredential | JwtVerifiableCredential`

Reply:
Yeah I noticed it wasn't used. Still I would make this change, because that is what people will use in their solutions. They have unsigned Credentials and signed (Verifiable) Credentials. It makes sense to have that distinction, even if we only accept Verifiable Credentials for now. I can imagine for instance that in a future version we would allow for a Credential to be passed into the callback (signing) mechanism

Reply:
I'll make the necessary changes